<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple TCP client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timerBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLCDNumber" name="timerNumber">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameNumberLabelLabel">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier 10 Pitch</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GAME N.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameNumberLabel">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier 10 Pitch</family>
          <pointsize>22</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="connectGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="hostLineEdit">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="conectBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="startGameBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>CHOOSE GAME</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="chooseGameList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chooseGameBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Choose Game</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startGameBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="talkGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>400</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="15" column="1">
       <widget class="QPushButton" name="sendBtn">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="msgLineEdit4"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="msgLineEdit"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="msgLineEdit3"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="msgLineEdit2"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label2City">
        <property name="text">
         <string>CITY</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label1Country">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>COUNTRY</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label4Animal">
        <property name="text">
         <string>ANIMAL</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label3Name">
        <property name="text">
         <string>NAME</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="infoLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>INFO PANEL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="msgsTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="letterLabelLabel">
        <property name="text">
         <string>CURRENT LETTER: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="letterLabel">
        <property name="font">
         <font>
          <family>Courier 10 Pitch</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="quitBtn">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>portSpinBox</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>conectBtn</tabstop>
  <tabstop>msgsTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
